docs/bots-guide.md

```diff
@@ -52,6 +49,10 @@ contrib_bots
         └───assets
            |
            └───pic.png
+├── bots_api
+│   ├── bot_lib.py
+│   ├── run.py
+├── integrations
```

Comment:
I decided to keep those out so that those reading the documentation as the first approach don't get confused or overwhelmed with the large number of files. Anyone who gets to the coding stage would notice the tests. Could replace it with the actual tree if you say so.